app/helpers/template_paths_helper.rb

```diff
@@ -0,0 +1,14 @@
+module TemplatePathsHelper
+  def template_path_for(type, route = 'path')
```

Comment:
Ok, I'll clean that. In general I agree with the rule unless I'm writing a helper... anyway to extend it from plugin you can create a concern that will alias the method and call the original one if it can't handle the type.